### PR TITLE
add Kapow v1.5.3

### PR DESCRIPTION
**Pull request by alliejp-tv**
Nice little time-tracking utility from the person who wrote [FocusWriter].

[FocusWriter]: https://github.com/caskroom/homebrew-cask/blob/master/Casks/focuswriter.rb

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

~~Additionally, if **updating a cask**:~~

- [ ] ~~`sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.~~

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/kapow.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'kapow' do
+  version '1.5.3'
+  sha256 '980ed8ec68045eb42c1d94a9fe809f50586a3b9a0248713aebf33ca62a37e625'
+
+  url 'https://gottcode.org/kapow/Kapow_1.5.3.dmg'
```
Comment (commitay) [inline Casks/kapow.rb]: `url "https://gottcode.org/kapow/Kapow_#{version}.dmg"`

File: Casks/kapow.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name 'Kapow'
+  homepage 'https://gottcode.org/kapow'
+
+  app 'Kapow.app'
+
+  zap trash: '~/Library/Application Support/GottCode/Kapow/*'
```
Comment (commitay) [inline Casks/kapow.rb]: `zap trash: '~/Library/Application Support/GottCode/Kapow'`

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/kapow.rb (hunk 2 above)
Comment (alfiepates) [reply Casks/kapow.rb]: Noticed and fixed this one, and proceeded to slightly break my PR by squashing the commits together >.<

**Review by commitay — APPROVED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**